safe-global/safe-wallet-web
# feat: Safe App info button

## What it solves

Resolves #1697

## How this PR fixes it

Preview drawers now only open the first time an app is opened. A new info button has been added to the Safe App cards that opens the preview drawer for later previews.

## How to test it

Open an app and observe that the preview drawer only opens the first time. In order to open it a second time, the info button needs to be clicked. Note; this does not persist across chains.

## Screenshots

![sidebar](https://user-images.githubusercontent.com/20442784/222377040-21220515-4852-4aa3-ae09-f5affaae08c1.gif)

![info-icon](https://user-images.githubusercontent.com/20442784/222377010-5a5861fa-c3a1-447f-908a-e1a8eab53f76.gif)

## Checklist
